app/models/lockstep/feature_flag.rb

```diff
@@ -0,0 +1,12 @@
+class Lockstep::FeatureFlag < Lockstep::ApiRecord
+  self.model_name_uri = "v1/feature-flags"
+  self.query_path=""
+  load_schema(Schema::FeatureFlag)
+
+  # Create method to override conventional create in lockstep/api_record.rb for sending attrs as an enclosed hash/object
```

Comment:
Can you please confirm the below list
1) the lockstep gem test cases are running as expected  without any failures ?
2) Please write testcases for the above changes
3) When accessed from INBOX app the changes work as expected
4) Please check in the inbox side by changing the version of lockstep-rails pointing to new version and check if any testcase is impacted by these changes

Reply:
I have tested the gem in all the scenarios no test cases are failing apart from the known ones.